test/unit/account/search_test.rb

```diff
@@ -153,4 +154,20 @@ class Account::SearchTest < ActiveSupport::TestCase
     result = provider.buyers.scope_search(created_within: ['2019-01-01', '2019-01-31'])
     assert_equal 2, result.size
   end
+
+  private
+
+  def index_finished?
+    # From https://freelancing-gods.com/thinking-sphinx/v3/testing.html
+    Dir[Rails.root.join(ThinkingSphinx::Test.config.indices_location, '*.{new,tmp}*')].empty?
+  end
+
+  def wait_until_index_finished
+    count = 0
+    until index_finished? || count >= 100
+      puts 'sphinx index not finished yet...'
+      count += 1
+      sleep 0.25
+    end
```

Comment:
imo here you should do a `puts` or something `unless index_finished?`, because that means that it stopped because `count >= 100` so if we get the failure again and we always see that puts, we have a clue 😄

Reply:
Done